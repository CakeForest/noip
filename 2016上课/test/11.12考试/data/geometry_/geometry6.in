10
70 73 27 10 94 92 40 68 56 28 
94 80 28 0 8 45 80 95 12 36 
10
24 2
3 2
20 45
29 76
15 40
20 9
40 68
54 50
92 75
81 20
