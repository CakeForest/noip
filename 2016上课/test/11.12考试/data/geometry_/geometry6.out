3
1
6
10
6
3
10
10
10
10
